src/Utility.ts

```diff
@@ -205,4 +205,13 @@ export namespace Utility {
             });
         });
     }
+
+    export function getJavaExecutable(): string {
+        let javaPath = '';
+        const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration('java');
+        if (config) {
+            javaPath = config.get<string>('home');
+        }
+        return javaPath + '/bin/java';
```

Comment:
Is `'/bin/java'` the correct default here, or would you want to return `'java'` if `!config`?

Reply:
You are absolutely right!